content/documentation/glossary/networking/index.adoc

```diff
@@ -0,0 +1,67 @@
+---
+title: "Networking"
+date: 2018-06-20T19:04:38+02:00
+draft: false
+type: "documentation/glossary"
+---
+
+= Kiali Networking Glossary
+:sectnums:
+:toc: left
+toc::[]
+:toc-title: Networking Glossary Content
+:keywords: Kiali Documentation Glossary
+:icons: font
+
+== Destination
+
+Read the <<Destination>> definition in link:https://istio.io/help/glossary/#destination[Istio Glossary]
+
+=== Destination Rule
+
+Read the <<Destination Rule>> definition in link:https://istio.io/docs/reference/config/istio.networking.v1alpha3/#DestinationRule[Istio Destination Rule]
+
+
+== Endpoint
+
+A communication endpoint is a type of communication network node. It is an interface exposed by a communicating party or by a communication channel
+
+== Error Rate
+
+It's the percentage the errors in the traffic to a specific object in a <<Rate Interval>>
+
+== Gateway
+
+Read the <<Gateway>> definition in link:https://istio.io/docs/reference/config/istio.networking.v1alpha3/#Gateway[Istio Gateway]
+
+== Inbound Metrics
+
+Incoming Metrics to a <<Workload>>, <<Service>> or <<Application>>
+
+== Outbound Metrics
+
+Outgoing Metrics to a <<Workload>> or <<Service>>
+
+== Port
+
+Read the <<Port>> definition in link:https://istio.io/docs/reference/config/istio.networking.v1alpha3/#Port[Istio Port]
+
+== Rate Interval
+
+It's an amount of time. By Default in Kiali last 10 minutes.
```

Comment:
Are we sure we want rate interval here?  Or just time range?  I thought rate interval was the bucket length.  Or maybe we need time range and rate interval?  @jotak probably can state whether it's OK as is.

Reply:
@jshaughn I think the term here in the Glossary refers to a specific label that is set in Service/Workload/Apps lists. It is indeed "Rate Interval" and is used to compute the error rate over this interval, nothing else.

But again (as per my previous comment) I think it would clarify if we write something like "It's the amount of time (or time interval?) over which a rate is computed". And I think there's no interest in writing Kiali default of 10 minutes.

Note to @jshaughn : practically, in these services/* list pages, the rate interval acts like a time range because we fetch only the last datapoint. Maybe the label could be renamed in the UI (but it was already discussed a while ago and afair we didn't find a better name).